deploy/package.json

```diff
@@ -5,7 +5,8 @@
   "main": "deploy.js",
   "scripts": {
     "compile": "pkg deploy.js --out-path build/tmp_build",
-    "lint": "eslint ."
+    "lint": "eslint .",
+    "deploy": "cd .. && npm run flatten && cd deploy && node deploy.js"
```

Comment:
could you consider to improve it by a way like this:
```
if [ ! "${NOFLAT}" == "true" ]; then cd ..; npm run flatten; cd deploy; fi && node deploy.js
```
and update the Docker file with `ENV NOFLAT "true"` ?

the rationale is the following: if the deployment is run within the docker container there is no need to flat contracts since they are already flattened. In most cases users will not re-compile contracts within the container so it is safe to assume that `flats` directory will not be updated without update entire docker image.

Reply:
Updated the script and Dockerfile https://github.com/poanetwork/tokenbridge-contracts/pull/334/commits/e3ef705908a29e0ef5a67a12d30bb58e633d9fcc

Reply:
cool!